src/pages/python/adding-alerts-and-slos.mdx

```diff
@@ -0,0 +1,57 @@
+import { Callout, Steps, Tabs, Tab } from "nextra-theme-docs";
+import AlertsSLOsLangIntro from "@/snippets/adding-alerts-slos.mdx";
+
+<AlertsSLOsLangIntro language="Python"/>
+
+## Pre-requisites
+
+1. Make sure you have setup and initialized Autometrics libraries as described in [Quickstart section](quickstart).
+
+2. Add the pre-configured Prometheus rules file to your Prometheus configuration. This file is located in the [Autometrics shared repository](https://github.com/autometrics-dev/autometrics-shared/blob/main/autometrics.rules.yml)
+
+## Usage
+
+<Steps>
+
+### Create `Objectives` with your desired SLOs
+
+To use autometrics SLOs and alerts, create one or multiple Objectives based on the function(s) success rate and/or latency, as shown below. 
+
+```python
+from autometrics import autometrics
+from autometrics.objectives import Objective, ObjectiveLatency, ObjectivePercentile
+
+API_SLO = Objective(
+    "random",
+    success_rate=ObjectivePercentile.P99_9,
+    latency=(ObjectiveLatency.Ms250, ObjectivePercentile.P99),
+)
+```
+
+### Add `Objectives` to the `autometrics` decorator
+The Objective (`API_SLO` in this case) can be passed as an argument to the
+autometrics decorator to include the given function in that objective.
+
+```python {10-12}
+from autometrics import autometrics
+from autometrics.objectives import Objective, ObjectiveLatency, ObjectivePercentile
+
+API_SLO = Objective(
+    "random",
+    success_rate=ObjectivePercentile.P99_9,
+    latency=(ObjectiveLatency.Ms250, ObjectivePercentile.P99),
+)
+
+@autometrics(objective=API_SLO)
+def api_handler():
+  # ...
+```
+
+You can group multiple functions into a single objective by passing the same objective to multiple functions.
```

Comment:
this might be worth stating earlier, then re-iterating in a comment inside the code block.